.devcontainer/Dockerfile

```diff
@@ -0,0 +1,5 @@
+ARG IMAGE="python:3.12"
```

Comment:
any reason not 3.13?

Reply:
we did discuss this before it was not available at the time of our discussion.
let me check if it's available now

Reply:
I think it's available now
https://github.com/devcontainers/images/tree/main/src/python 

let me test it tonight.

Reply:
they are lying the only available 3.13 is dev-3.13.
let's keep 3.12 until they release a stable 3.13